### PR TITLE
Ben/v3 vaccine form validation

### PR DESCRIPTION
**Pull request by bionara**
# Description

Provides conditional validation on the vaccine form's name and description fields
Also adds sme rather useful yarn commands..!

## On what platform have you tested the change?

- [ ] Android - Emulator
- [ ] Android - Physical device
- [x] iOS - Emulator
- [ ] iOS - Physical device

## Screenshots

_Please attach screenshots showing the change if relevant_

## Checklist

- [ ] I have updated mockServer
- [ ] I've added analytics as relevent
- [ ] I have run `npm test`
- [ ] I have run `npm run test:i18n`

## Out of scope and potential follow-up

_Is there any related changes that you plan to do in a follow-up PR or voluntarily excluded from the scope?_

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -4,7 +4,9 @@
   "scripts": {
     "start": "react-native start",
     "start:reset": "react-native start --reset-cache",
+    "build-fonts": "./.circleci/pull_private_assets.sh",
     "android": "ENVFILE=.env.staging react-native run-android --variant=debug --appId=com.joinzoe.covid_zoe.qa",
+    "android:local": "ENVFILE=.env.staging react-native run-android --variant=debug --appId=com.joinzoe.covid_zoe.qa",
```
Comment (petermason) [inline package.json:9]: duplicate command

Comment (bionara) [reply package.json:9]: Good catch. changed the file to .env.local thx!

**Comment by bionara** [comment]
I've set this as a draft as after pulling in develop i cannot build and truly sign off the work vs develop - it did work before i pulled in develop, but that sounds like a developer's famous last words if i ever did hear them...!

File: src/features/vaccines/fields/VaccineDoseQuestion.tsx [hunk 2]
```diff
@@ -172,14 +172,33 @@ const styles = StyleSheet.create({
...
+      secondBrand: Yup.string().when(['secondDoseDate', 'secondDescription'], {
+        is: [!undefined, undefined],
+        then: Yup.string().required(i18n.t('validation-error-please-select-option')),
+      }),
+      secondDescription: Yup.string().when(['secondDoseDate', 'secondBrand'], {
+        is: [!undefined, 'not_sure'],
+        then: Yup.string().required(i18n.t('validation-error-please-select-option')),
+      }),
+    },
+    // These are to flag against circular dependency errors:
```
Comment (selvach) [inline src/features/vaccines/fields/VaccineDoseQuestion.tsx:198]: Wow... TIL

**Review by selvach — APPROVED** [review]

**Pull request merged**